server/protocols/jmap/src/main/java/org/apache/james/jmap/model/VacationResponse.java

```diff
@@ -71,7 +71,7 @@ public Builder textBody(String textBody) {
         }
 
         public Builder fromVacation(Vacation vacation) {
-            this.id = vacation.getId();
+            this.id = Vacation.ID;
```

Comment:
I do not store the id in Cassandra as it always have value "singleton" saving space. So the vacation object stored do not need an Id field, whereas the vacationResponse does have one. VacationResponses are then directly granted the "singleton" id, the only possible value.... 
I need to affect a value here... 
What I will do is remove the id field from the builder and affect it during build